ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -68,10 +69,40 @@ interface Props {
     onResume?: () => void;
 }
 
+const linkify = new LinkifyIt();
+
+function linkifyText(text: string) {
```

Comment:
This should probably be a util function given that it's likely to be used in other places.

That also raises the question: is this the only place we'd add this? Does this open too many floodgates?

Reply:
No you're right we could definitely use this in other places, can't see where atm but I'll make it easily reusable